templates/web/pages/about.tmpl

```diff
@@ -0,0 +1,142 @@
+{{ define "about" }}
+{{ template "header" . }}
+
+<section class="hero-head">
+	{{ template "navbar" }}
+	</div>
+	<div class="hero-body">
+		<div class="container has-text-centered">
+			<h1 class="title is-1 is-spaced">About BoilerMake</h1>
+		</div>
+	</div>
+</section>
+
+<section class="section">
+	<div class="container">
+		<div class="columns is-centered content">
+			<div class="column is-7">
+				<p>BoilerMake is a hackathon that brings over 500 hackers
+					from across the country under one roof for one incredible weekend. As a team,
+					our mission is to empower each and every hacker to build anything that they can imagine,
+					we’ll pull out all the stops to make sure that dream becomes a reality.
+					We find happiness in knowing that every hacker will walk out of our doors with memories
+					that will last a lifetime.</p>
+
+			</div>
+		</div>
+	</div>
+</section>
+
+<section class="section">
+	<div class="container">
+		<div class="columns is-mobile">
+			<div class="column">
+				<img src='{{ static_path "about_group_2.jpg" }}' />
+			</div>
+		</div>
+	</div>
+</section>
+
+<section class="section">
+	<div class="container">
+		<div class="columns is-centered">
+			<div class="column is-6">
+				<p>This year, 24 students from Purdue University have come
+					together to host an unforgettable hackathon. All the work that has gone into planning
+					this event—everything from setting up the venue to the annoying emails that we berate
+					your inboxes with—was accomplished by somebody just like you. </p>
+
+				<p> <br> If this sort of stuff sounds like your thing, stay tuned for more updates!
+					We’ll be looking for new people to help us throw another successful hackathon in
+					2018! As always - if you have any questions, feel free to hit us up at team@boilermake.org! </p>
+			</div>
+		</div>
+	</div>
+</section>
+
+<section class="section">
+	<h1 class="title is-1 is-spaced has-text-centered">History</h1>
+	<div class="columns is-mobile is-vcentered content">
```

Comment:
Each of these history entries should also be wrapped in a `<div class="container">...</div>` so that their width is limited on high res displays